superset/assets/visualizations/main.js

```diff
@@ -51,53 +51,56 @@ export const VIZ_TYPES = {
+  [VIZ_TYPES.directed_force]: () => loadVis(import(/* webpackChunkName: "directed_force" */ './directed_force.js')),
+  [VIZ_TYPES.chord]: () => loadVis(import(/* webpackChunkName: "chord" */ './chord.jsx')),
+  [VIZ_TYPES.dist_bar]: () => loadVis(import(/* webpackChunkName: "nvd3_vis" */ './nvd3_vis.js')),
+  [VIZ_TYPES.filter_box]: () => loadVis(import(/* webpackChunkName: "filter_box" */ './filter_box.jsx')),
+  [VIZ_TYPES.heatmap]: () => loadVis(import(/* webpackChunkName: "heatmap" */ './heatmap.js')),
+  [VIZ_TYPES.histogram]: () => loadVis(import(/* webpackChunkName: "histogram" */ './histogram.js')),
+  [VIZ_TYPES.horizon]: () => loadVis(import(/* webpackChunkName: "horizon" */ './horizon.js')),
+  [VIZ_TYPES.iframe]: () => loadVis(import(/* webpackChunkName: "iframe" */ './iframe.js')),
+  [VIZ_TYPES.line]: () => loadVis(import(/* webpackChunkName: "nvd3_vis" */ './nvd3_vis.js')),
+  [VIZ_TYPES.time_pivot]: () => loadVis(import(/* webpackChunkName: "nvd3_vis" */ './nvd3_vis.js')),
+  [VIZ_TYPES.mapbox]: () => loadVis(import(/* webpackChunkName: "mapbox" */ './mapbox.jsx')),
+  [VIZ_TYPES.markup]: () => loadVis(import(/* webpackChunkName: "markup" */ './markup.js')),
+  [VIZ_TYPES.para]: () => loadVis(import(/* webpackChunkName: "parallel_coordinates" */ './parallel_coordinates.js')),
+  [VIZ_TYPES.pie]: () => loadVis(import(/* webpackChunkName: "nvd3_vis" */ './nvd3_vis.js')),
+  [VIZ_TYPES.pivot_table]: () => loadVis(import(/* webpackChunkName: "pivot_table" */ './pivot_table.js')),
+  [VIZ_TYPES.sankey]: () => loadVis(import(/* webpackChunkName: "sankey" */ './sankey.js')),
+  [VIZ_TYPES.separator]: () => loadVis(import(/* webpackChunkName: "markup" */ './markup.js')),
+  [VIZ_TYPES.sunburst]: () => loadVis(import(/* webpackChunkName: "sunburst" */ './sunburst.js')),
+  [VIZ_TYPES.table]: () => loadVis(import(/* webpackChunkName: "table" */ './table.js')),
+  [VIZ_TYPES.time_table]: () => loadVis(import(/* webpackChunkName: "time_table" */ './time_table.jsx')),
+  [VIZ_TYPES.treemap]: () => loadVis(import(/* webpackChunkName: "treemap" */ './treemap.js')),
+  [VIZ_TYPES.country_map]: () => loadVis(import(/* webpackChunkName: "country_map" */ './country_map.js')),
+  [VIZ_TYPES.word_cloud]: () => loadVis(import(/* webpackChunkName: "word_cloud" */ './word_cloud.js')),
+  [VIZ_TYPES.world_map]: () => loadVis(import(/* webpackChunkName: "world_map" */ './world_map.js')),
+  [VIZ_TYPES.dual_line]: () => loadVis(import(/* webpackChunkName: "nvd3_vis" */ './nvd3_vis.js')),
+  [VIZ_TYPES.event_flow]: () => loadVis(import(/* webpackChunkName: "EventFlow" */ './EventFlow.jsx')),
+  [VIZ_TYPES.paired_ttest]: () => loadVis(import(/* webpackChunkName: "paired_ttest" */ './paired_ttest.jsx')),
+  [VIZ_TYPES.partition]: () => loadVis(import(/* webpackChunkName: "partition" */ './partition.js')),
+  [VIZ_TYPES.deck_scatter]: () => loadVis(import(/* webpackChunkName: "deckgl/layers/scatter" */ './deckgl/layers/scatter.jsx')),
```

Comment:
can we use the same `webpackChunkName` for all `deck_*`?

Reply:
What's the goal here? for readability or to have all of that code in one chunk? 

I think using the same chunk name would only work if the same code was imported which is currently not possible since there are multiple index files for the various deck gl layers, right?